extension/podstation.css

```diff
@@ -246,3 +246,68 @@ img.listItemImageSmall {
 #aboutContent {
 	font-size: medium;
 }
+
+.dark-scheme{
+	--dark_text: #d1d2d3;
+	--dark_bg : #333333; /* 1e1e1e*/
+	--dark_item_bg :#1e1e1e;
+	--dark_head : #3d3d3d;
+	--dark_border : #565856;
+	--dark_entrybox: #2e2e2e; /* #454545; */
+}
+
+body.dark-scheme{
+	background-color: #1e1e1e;
+	color:var(--dark_text)
+}
+
+.dark-scheme #playlist{
+	background-color: var(--dark_head);
+}
+
+.dark-scheme .playlistEntry{
+	border-bottom-color:var(--dark_border);
+}
+
+.dark-scheme .itemHeader, .dark-scheme .miniPlayer{
+	background: var(--dark_head);
+}
+
+.dark-scheme .miniPlayer{
+	box-shadow: 0px 0px 5px var(--dark_text);
+}
+
+.dark-scheme .episodeEntry, .dark-scheme .listEntry, .dark-scheme .podcastEntry{
+	border:none;
+	box-shadow: 0 -2px 10px rgba(0, 0, 0, 1);
+	background:var(--dark_entrybox);
```

Comment:
can you add a 0.5s transition here and on the hover?
It will look weird when transition from/to dark theme, but the mouse hover will be smoother.

Reply:
I have added transition for moving from/to dark theme, also for the hover.
Do you think we can also try the transition hover effect for the white theme So that it looks consistent across both the themes?

Reply:
Let's keep the hover effect only on the dark mode, I am already too used with current style of the light one :D

Reply:
@dellagustin Sure. I haven't added hover effect for the light theme,  so we are good with the dark theme, you can test it and merge it.